### ISSUE TITLE
Projects

### ISSUE THREAD

**Issue by controversial** [thedillonb/CodeHub#228]
Once the Projects API stabilizes, will CodeHub support it? I see a great potential here for a nice touch interface, push notifications, etc, just like the Trello app has.

**Comment by thedillonb** [comment]
Absolutely. I'm personally pumped about Projects so you can be assured CodeHub will support it!

**Comment by controversial** [comment]
Awesome. Glad this is actively maintained

**Comment by paolokivra** [comment]
Any news about this?

**Comment by streerwise702** [comment]
No but I have a question abouT IOS11 on windows phone basically some one is using codes to lock me out of my iCloud is there any code for fixing the issue

**Comment by jll66** [comment]
On Sep 25, 2017 3:47 AM, paolo <notifications@github.com> wrote:Any news about this?

**Comment by garrettw** [comment]
@streerwise702 Your problem has nothing to do with CodeHub or this particular thread -- nor was paolokivra talking to you. Also there is no such thing as "iOS 11 on Windows Phone".
Please refrain from commenting things unrelated to the CodeHub app.

**Comment by DavidHaslam** [comment]
Any news on supporting projects?

**Comment by melMass** [comment]
@thedillonb When do you plan to add projects support, please?
I'm starting to rely on it a lot for some projects.
